packages/server-ct/src/routes-ct.ts

```diff
@@ -3,12 +3,12 @@ import { ErrorRequestHandler, Express } from 'express'
 import httpProxy from 'http-proxy'
 import send from 'send'
 import { NetworkProxy } from '@packages/proxy'
-import { handle, serve, serveChunk } from '@packages/runner-ct'
+import { handle, serve, serveChunk } from './runner-ct'
 import xhrs from '@packages/server/lib/controllers/xhrs'
 import { SpecsStore } from '@packages/server/lib/specs-store'
-import staticPkg from '@packages/static'
 import { ProjectCt } from './project-ct'
 import { ProjectE2E } from '../../server/lib/project-e2e'
+import { getPathToDist } from '@packages/resolve-dist'
```

Comment:
Should `@packages/resolve-dist` be added to `packages/server-ct/package.json`?

Reply:
I don't think it should matter, but I suppose so to be consistent